src/coreclr/jit/gentree.cpp

```diff
@@ -12093,7 +12093,14 @@ void Compiler::gtDispTree(GenTree*     tree,
                 case NI_System_Runtime_CompilerServices_RuntimeHelpers_IsKnownConstant:
                     printf(" isKnownConst");
                     break;
-
+#if defined(FEATURE_SIMD)
+                case NI_SIMD_UpperRestore:
+                    printf(" simd_upperRestr");
```

Comment:
nit: 1 extra character to give the full name:
```suggestion
                    printf(" simdUpperRestore");
```